bookwyrm/activitypub/base_activity.py

```diff
@@ -73,7 +73,7 @@ class ActivityObject:
     def __init__(
         self,
         activity_objects: Optional[list[str, base_model.BookWyrmModel]] = None,
```

Comment:
Is `activity_objects` a list, or a dict? From the typing syntax and looking at `activitypub/__init__.py`, it seems like a dict; but I'm very new to this code.

Reply:
@dato Nice catch, that's indeed incorrect. I'm also still quite new to this code, but trying to add type annotations helps a lot in getting to know it ;)

@hughrun Thanks! That's very helpful to know :)